app/src/androidTest/java/ch/epfl/sweng/swenggolf/MainMenuActivityInstrumentedTestAvailable.java

```diff
@@ -59,6 +63,19 @@ public void canCloseAndOpenDrawerWithButton() {
         onView(v).perform(close());
         onView(v).check(matches(isClosed(Gravity.LEFT)));
     }
+
+    @Test
+    public void rotationDoesntYieldANewFragment() throws InterruptedException {
+        List<Fragment> currents = mMenuRule.getActivity()
+                .getSupportFragmentManager().getFragments();
+        int sizeBefore = currents.size();
+        mMenuRule.getActivity()
+                .setRequestedOrientation(ActivityInfo.SCREEN_ORIENTATION_LANDSCAPE);
+        Thread.sleep(500);
```

Comment:
Code Climate doesn't seem to like your Thread.sleep!

Reply:
Maybe we can leave it this way now and I'll try to find another way one of these days.